src/components/button-minor/button-minor.test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ButtonMinor from ".";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+
+test("should render with children", () => {
+  render(<ButtonMinor>foo</ButtonMinor>);
+
+  expect(screen.getByRole("button", { name: "foo" })).toBeVisible();
+});
+
+test("should render disabled button when `disabled` is true", () => {
+  render(<ButtonMinor disabled>foo</ButtonMinor>);
+
+  expect(screen.getByRole("button", { name: "foo" })).toBeDisabled();
+});
+
+// coverage
```

Comment:
**comment (non-blocking)**: I'm not a fan of putting a `coverage` comment on all these tests. While I'm sure they are needed for coverage, they're also testing the styles of various button types etc, and that's not without value. I think the tests are fine and I'd just remove these comments.

Reply:
I think we started adding these to let reviewers know that these are needed for coverage as we had to clarify during reviews, specially with these style tests. These might also be useful once we have combined coverage with PW to know which tests can be removed/should be done in PW/chromatic.

Reply:
Though this is something that could be discussed with the team as at the moment I think this is mostly preference.